Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:alpine
+
+EXPOSE 6667/tcp 6697/tcp 8080/tcp
+
+RUN \
+    apk add --update git && \
+    rm -rf /var/cache/apk/*
+
+RUN mkdir -p /go/src/github.com/oragono/oragono
+WORKDIR /go/src/github.com/oragono/oragono
+
+COPY . /go/src/github.com/oragono/oragono
+
+RUN go get -v -d
+RUN go build .
```

Comment:
I'm alright with a `go build oragono.go`, since that gomatic enough imo. But yeah, I've changed the whole `go get` and all as above, in the proper repo.